src/pages/Tournament/TournamentDetails/AdminManagementPanel/parts.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { Button, Modal, Select } from 'antd';
+
+import { ClosableComponentProps } from 'utils/interfaces';
```

Comment:
Codacy has a fix for the issue: [Replace `{·ClosableComponentProps·}·from·'utils/interfaces` with `React,·{·useState·}·from·'react`](https://app.codacy.com/gh/othello-organisation-singapore/othello-storm-system-frontend/pullRequest?prid=8062249)
```suggestion
import React, { useState } from 'react';
```